color/src/impl_bytemuck.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 the Color Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![allow(unsafe_code, reason = "unsafe is required for bytemuck unsafe impls")]
+
+use crate::{AlphaColor, ColorSpace, OpaqueColor, PremulColor, Rgba8};
+
+unsafe impl<CS: ColorSpace> bytemuck::Pod for AlphaColor<CS> {}
+
+unsafe impl<CS: ColorSpace> bytemuck::Zeroable for AlphaColor<CS> {}
+
+unsafe impl<CS: ColorSpace> bytemuck::Pod for OpaqueColor<CS> {}
+
+unsafe impl<CS: ColorSpace> bytemuck::Zeroable for OpaqueColor<CS> {}
+
+unsafe impl<CS: ColorSpace> bytemuck::Pod for PremulColor<CS> {}
+
+unsafe impl<CS: ColorSpace> bytemuck::Zeroable for PremulColor<CS> {}
+
+unsafe impl bytemuck::Pod for Rgba8 {}
+
+unsafe impl bytemuck::Zeroable for Rgba8 {}
```

Comment:
Can we have a "test" that all fields of these types (i.e. by doing destructuring) are either `std::marker::PhantomData` or are themselves Pod?

Reply:
I've done something here.